contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -98,6 +98,7 @@ extern bool pltsql_recursive_triggers;
 extern bool restore_tsql_tabletype;
 extern bool babelfish_dump_restore;
 extern bool pltsql_nocount;
+extern const char *ATTOPTION_BBF_ORIGINAL_NAME;
```

Comment:
Can you show me a a const define case using Upper naming only ?

Reply:
This is an existing variable I am using